frontend/src/pages/NewRunV2.test.tsx

```diff
@@ -302,34 +469,65 @@ describe('NewRunV2', () => {
                 relationship: ApiRelationship.OWNER,
               },
               {
-                key: { id: TEST_PIPELINE_VERSION_ID, type: 'PIPELINE_VERSION' },
-                relationship: 'CREATOR',
+                key: {
+                  id: TEST_PIPELINE_VERSION_ID,
+                  type: ApiResourceType.PIPELINEVERSION,
+                },
+                relationship: ApiRelationship.CREATOR,
               },
             ],
             service_account: '',
           }),
         );
       });
     });
-  });
 
-  describe('cloning a existing run', () => {
-    it('only shows clone run name from original run', () => {
-      const getRunSpy = jest.spyOn(Apis.runServiceApi, 'getRun');
-      getRunSpy.mockResolvedValue(API_RUN_DETAILS);
+    it('submits a run (clone SDK-created run)', async () => {
+      const createRunSpy = jest.spyOn(Apis.runServiceApi, 'createRun');
+      createRunSpy.mockResolvedValue(API_SDK_RUN_DETAILS_CLONE);
+
       render(
         <CommonTestWrapper>
           <NewRunV2
             {...generatePropsClonedRun()}
-            existingRunId='e0115ac1-0479-4194-a22d-01e65e09a32b'
-            apiRun={API_RUN_DETAILS}
+            existingRunId={TEST_RUN_ID}
+            apiRun={API_SDK_RUN_DETAILS_CREATE}
```

Comment:
The name `API_SDK_RUN_DETAILS_CREATE` is a bit confusing. Can we just call it `API_SDK_CREATED_RUN_DETAIL`?